docs/blog/2019-05-25-sentry-integration-with-gatsby/index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Integrating Sentry Into Your Gatsby Project"
+date: "2019-05-25"
+author: "Jonathan Reeves"
+tags:
+  - error monitoring
+  - post
+  - sentry
+---
+
+## Assuming knowledge on my part
+
+I'm going to do something that you are told not to ever do and assume that you already have a project created and are just looking to install an error tracking software to help you fix and resolve errors before any users find it or at least before majority of your users find it and start complaining.
+
+## What is Sentry?
+
+Are you tired of waiting for users to find and report errors with your application or website? Then look no further than Sentry.
+
+Sentry is an open-source error tracking library that helps developers monitor and fix crashes in real time. Sentry is iterating continuousy to boost efficiency and improve user experience. No begs the questions how easy is the setup? And what will it cost me? I'm going to answer the second question first, ready? Sentry has a free tier you can start for free right now. The plans are pay as you go. The free tier is most likely all you will need at first. But after your user base grows you might want to consider upgrading. They don't require you to fill out a contract, you can upgrade or downgrade at any time. Rates will be automatically adjusted at the beginning of your next billing cycle. How awesome is that?
+
+Now on to the point of this post. How do you install the wonderful software with your Gatsby project. The next few steps are going to show you how to go about doing that. Are you ready? Great let's get started.
+
+## Installation and directory structure
+
+Step 1. Head over to the Sentry website to signup for your free tier. If you have an Azure DevOps account you can use that if not I recommend just using your GitHub account. But if you would rather it be tied to say your email then by all means use that. See below image of signup page.
+
+![Signup Page](./images/sentry-signup-page.png)
+
+Step 2. Now that you are setup with an account and logged in you should be taken to a page that has a navigation on the left with projects, issues, etc. See below screenshot
+
+![Projects Page Example](./images/projects-page.png)
+
+Step 3. I am going to be using React in this tutorial. We are now going to install the Sentry SDK as a dependency using yarn or npm. You will need to use @sentry/browser as well which we will setup below.
+
+```shell
+# Using yarn
+$ yarn add @sentry/browser
+
+# Using npm
+$ npm install @sentry/browser
+```
+
+## React code to connect to Sentry
+
+After installing Sentry you will be given a value which has been dubbed a DSN or Data Source Name. It looks a lot like a standard URL, but it's just a representation of the configuration required by the Sentry SDKs. It consists of several things including the protocol, public key, the server address, and the project identifier.
```

Comment:
this is step 4...

Reply:
Added Step 4. text at the beginning.

Reply:
renumber also the other steps

Reply:
but i think maybe there is no need for the `step xx` in the text, because the headers guides the user through the steps